### PR TITLE
Jwplayer Event Bus (for tracking purposes)

### PR DESCRIPTION
**Pull request by bkoval**
@fraszczakszymon @Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+		}
+	}
+
+	function handleTime(prefix, data) {
+		var positionFloor = Math.floor(data.position),
+			percentPlayed = Math.floor(positionFloor * 100 / data.duration);
+
+		if (positionFloor > state[prefix].progress.durationWatched && positionFloor % 5 === 0) {
+			playerInstance.trigger(prefix + 'SecondsPlayed', { value: positionFloor });
+			console.log(prefix, positionFloor);
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js]: MAJOR Remove this logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	}
+
+	function getNewState() {
+		return {
+			ad: getDefaultState(),
+			video: getDefaultState()
+		}
+	}
+
+	function handleTime(prefix, data) {
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js:31]: MAJOR Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js [hunk 3]
```diff
@@ -66,66 +49,46 @@ define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function
 
 			var relatedPlugin = playerInstance.getPlugin('related');
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js]: MAJOR Remove the declaration of the unused 'relatedPlugin' variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
MAJOR Remove this useless assignment to local variable "relatedPlugin" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by fraszczakszymon — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**